pages/signup.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import Card from '../components/Card'
-import Layout from '../components/Layout'
+import LandingNav from '../components/LandingNav'
 
 const Signup: React.FC = () => {
   return (
-    <Layout>
+    <div>
+      <LandingNav />
```

Comment:
Why did you removed `<Layout />`
Layout should wrap all pages (except the landing page)

Reply:
No. The Login / Signup page share the `Nav` with the landing page,. I changed `Layout` to use the Nav for courses. My implementation makes Layout uses the `secondNav` which isn't compatible with signup and login.